test/libsolidity/smtCheckerTests/types/type_minmax.sol

```diff
@@ -82,4 +82,18 @@ contract C {
 // SMTEngine: all
 // ----
 // Warning 6328: (178-207): CHC: Assertion violation happens here.
-// Info 1391: CHC: 24 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
+// Warning 6328: (872-900): CHC: Assertion violation might happen here.
+// Warning 6328: (942-970): CHC: Assertion violation might happen here.
```

Comment:
Why do we have all those `might happen` here? It looks liek it shouldn't be reported.